admin/create-osx-user.sh

```diff
@@ -43,5 +43,6 @@ dscl . create /Users/${USERNAME} NFSHomeDirectory /Users/${USERNAME}
 cp -R /System/Library/User\ Template/English.lproj /Users/${USERNAME}
 chown -R ${USERNAME}:${GROUP_NAME} /Users/${USERNAME}
 
+echo "${USERNAME}  ALL=(ALL:ALL) ALL" >> /etc/sudoers
```

Comment:
Shoud we check if this line is not in the file before writing into it?

Something like:

```
SUDO_USER="$USERNAME  ALL=(ALL:ALL) ALL"
grep -q -F 'include "/etc/sudoers"' $SUDO_USER  || echo '$SUDO_USER"' >> /etc/sudoers
```